core/src/toga/widgets/base.py

```diff
@@ -33,6 +54,20 @@ def __init__(
         :param style: A style object. If no style is provided, a default style
             will be applied to the widget.
         """
+        # if the object has _USE_DEBUG_BACKGROUND=True and layout debug mode is on, change bg color
+        if hasattr(self, "_USE_DEBUG_BACKGROUND") and self._USE_DEBUG_BACKGROUND:
+            if "TOGA_DEBUG_LAYOUT" in environ and environ["TOGA_DEBUG_LAYOUT"] == "1":
+                if Widget._debug_color_index == len(debug_background_palette) - 1:
+                    Widget._debug_color_index = 0
+                else:
+                    Widget._debug_color_index += 1
+                style = style if style else Pack()
+                style.background_color = debug_background_palette[
+                    Widget._debug_color_index
+                ]
```

Comment:
```suggestion
                Widget._debug_color_index += 1
                style = style if style else Pack()
                style.background_color = debug_background_palette[
                    Widget._debug_color_index % len(debug_background_palette)
                ]
```

Reply:
I don't *think* it would make a difference here, since one blank Pack should be the same as any other, but checking `style is not None` would be more consistent with other things that will interact with future versions of Travertino...